javascript-functions-practice/get-first-element.js

```diff
@@ -0,0 +1,5 @@
+/* exported getFirstElement */
+function getFirstElement([a, b, c, d]) {
+  var firstElement = [a, b, c, d];
```

Comment:
You should just have a parameter named `array`. What if your array argument has more than four indexes?